### ISSUE TITLE
Scan not starting, missing scanners

### ISSUE THREAD

**Issue by LoZio** [immauss/openvas#43]
Using latest as of today (21.04.1), two different machines.
First startup took a couple of hours each (not expected based on previous experience).
Once it update it all, I started several scans. All do not give results.
Trying to debug the OpenVAS part it seems that no scanners are selected.
Going to the scanner setup in the scan profile, I see something is missing here...
![image](https://user-images.githubusercontent.com/869890/120209021-76105c80-c22e-11eb-9be8-628a7dac73eb.png)
inested of this, from a working setup
![image](https://user-images.githubusercontent.com/869890/120209750-53cb0e80-c22f-11eb-8087-33489f828b18.png)

So basically, no scanners => no scan.
Thanks

**Comment by immauss** [comment]
Ugh ... sounds like something went wrong with the new db generation. 
for now, you should be able to force the 21.04.01 to use the slight out of data data while I figure out what happend to the current "latest" tag image.

Thanks,
Scott

**Comment by immauss** [comment]
OK ... it looks like a forgot to update the tags in the scripts to do the weekly update. I've fixed that ... and I'm now waiting for the rebuild to complete... 
I'll let you know once I've tested ...

**Comment by immauss** [comment]
Well .... as it turns out ... the refresh process needs about 5G of space to pull all the feeds, rebuild the database and then compress the pieces. The machine running that ... did not have 5G of space. So it failed out and made a mess. Please give the "latest" a another shot and let me know if you have any issues. 

-Scott

**Comment by immauss** [comment]
Oh ... well that is curious ...
Mine is the same way. 
I'm looking into it. 

Thanks,
Scott

**Comment by immauss** [comment]
Actually ... it looks like the feed has just not been updated. 
I did a manual pull of the feed, and there were no updates. I suppose that is not unusual, but I'll ask in the community forum to just make sure. 

Thanks,
Scott

**Comment by LoZio** [comment]
I just checked my manually compiled and updated version ad it seems you're right. I updated it on monday but that was not updated in some time and gave me new data.
![image](https://user-images.githubusercontent.com/869890/120595273-62d5da80-c442-11eb-992e-e9f454ccaaf3.png)
Should I open a different issue for the unhealthy status? Just to avoid messing things...
Thanks

**Comment by immauss** [comment]
So it is the correct behavior.

https://community.greenbone.net/t/gvmd-data-too-old-34-days-please-check-the-automatic-synchronization-of-your-system/7565/10
